### PR TITLE
Show trust variable deprecation warning only if used

### PR DESCRIPTION
**Pull request by tonistiigi**
At the moment warning is always shown if you just try DCT for the first time.

Signed-off-by: Tonis Tiigi tonistiigi@gmail.com

cc @diogomonica

### CODE REVIEW

**Comment by diogomonica** [comment]
LGTM

File: api/client/trust.go [hunk 1]
```diff
@@ -199,14 +199,16 @@ func (cli *DockerCli) getPassphraseRetriever() passphrase.Retriever {
 	// Backwards compatibility with old env names. We should remove this in 1.10
 	if env["root"] == "" {
 		env["root"] = os.Getenv("DOCKER_CONTENT_TRUST_OFFLINE_PASSPHRASE")
-		fmt.Fprintf(cli.err, "[DEPRECATED] The environment variable DOCKER_CONTENT_TRUST_OFFLINE_PASSPHRASE has been deprecated and will be removed in v1.10. Please use DOCKER_CONTENT_TRUST_ROOT_PASSPHRASE\n")
-
+		if os.Getenv("DOCKER_CONTENT_TRUST_OFFLINE_PASSPHRASE") != "" {
```
Comment (icecrime) [inline api/client/trust.go]: Can we avoid calling `os.Getenv` twice on the same key?

Comment (tonistiigi) [inline api/client/trust.go]: Updated. Changed for these if clauses, didn't change rest of the file because that is not what this PR fixes.

**Comment by runcom** [comment]
LGTM

**Pull request merged**